django_cookies_samesite/user_agent_checker.py

```diff
@@ -70,22 +70,28 @@ def is_chrome_supported_version(self):
         return True if self.is_chrome_browser() and (
             self.BUGGY_CHROME_VERSION_MAJOR_MIN > uav or uav > self.BUGGY_CHROME_VERSION_MAJOR_MAX) else False
 
-    def is_supported_chrome(self):
-        return self.is_chrome_supported_version()
+    def get_user_agent_os_version(self, version_type):
+        return int(self.user_agent_os.get(version_type, '0') if self.user_agent_os.get(version_type) else '0')
```

Comment:
Could this be simplified to `self.user_agent_os.get(version_type) or '0'`?

Reply:
@timmc-edx  @jotes created PR #31